tools/coverage.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+COVERAGE_DIR="${GOPATH}/src/github.com/YAWAL/GetMeConf/tools"
+
+COVERAGE_DIR="${COVERAGE_DIR:-coverage}"
```

Comment:
COVERAGE_DIR - on line 3 and 5.
Could be 
COVERAGE_DIR="${GOPATH}/src/github.com/YAWAL/GetMeConf/tools:-coverage}"
?